nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -572,41 +580,34 @@
   The derivation now installs "impl" headers selectively instead of by a wildcard.
   Use `imgui.src` if you just want to access the unpacked sources.
 
-- Linux 4.19 has been removed because it will reach its end of life within the lifespan of 24.11
+- Linux 4.19 has been removed because it will reach its end of life within the lifespan of 24.11.
 
 - Unprivileged access to the kernel syslog via `dmesg` is now restricted by default. Users wanting to keep an
   unrestricted access to it can set `boot.kernel.sysctl."kernel.dmesg_restrict" = false`.
 
 - The `i18n.inputMethod` module introduces two new properties:
   `enable` and `type`, for declaring whether to enable an alternative input method and defining which input method respectfully. The options available in `type` are the same as the existing `enabled` option. `enabled` is now deprecated, and will be removed in a future release.
 
-- `security.pam.u2f` now follows RFC42.
-  All module options are now settable through the freeform `.settings`.
-
-- Mikutter was removed because the package was broken and had no maintainers.
-
-- Gollum was upgraded to major version 6. Read their [migration notes](https://github.com/gollum/gollum/wiki/6.0-Release-Notes).
+- `security.pam.u2f` now follows RFC42; all module options are now configurable through `security.pam.u2f.settings`.
```

Comment:
It might be good to link [RFC 42](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md).

Reply:
I'm actually not really sure how we should go about these since there's a few mentions of it. IMO any mention of RFC42 could be removed in favor of just referring to them as freeform modules, since users will probably be more familiar with that concept (given RFCs are more of a developer oriented thing)

Would like some feedback on this, though